client/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+        <div id="Photo-gallery"></div>
+        <div id="Checkout"></div>
+        <div id="Reviews"></div>
+        <div id="Recommendations"></div>
+        <div id="experiences"></div>
+        <script src="https://unpkg.com/react@16/umd/react.development.js"></script>
+        <script src='https://unpkg.com/react-dom@16/umd/react-dom.development.js'></script>
+        <!-- <script src="htttp://localhost:3001/app.js"></script>
```

Comment:
My module renders based on a listing ID, and the import path should be `src="http://localhost:3001/:listingid/bundle.js"`